Library/Homebrew/linkage_checker.rb

```diff
@@ -78,8 +79,8 @@ def broken_library_linkage?(test: false, strict: false)
 
     issues = [@broken_deps, @broken_dylibs]
     if test
-      issues += [@unwanted_system_dylibs, @version_conflict_deps]
-      issues += [@undeclared_deps, @files_missing_rpaths, @executable_path_dylibs] if strict
+      issues += [@unwanted_system_dylibs, @version_conflict_deps, @indirect_deps, @undeclared_deps]
```

Comment:
We should probably keep `@undeclared_deps` and `@indirect_deps` under `strict` for now, because their failures will make `brew` attempt a local source build after attempting a bottle install that fails these linkage tests

Reply:
@carlocab updated this to add/keep them behind strict.

Reply:
Not seeing the update. Did you forget to push? My connection is poor at the moment tho, so I may still be seeing an old version.

Reply:
@carlocab Had bad airport internet, pushed now!